bitnami/mean/templates/NOTES.txt

```diff
@@ -1,23 +1,23 @@
 
 1. Get the URL of your MEAN app by running:
 
-{{- if contains "NodePort" .Values.serviceType }}
+{{- if contains "NodePort" .Values.service.type }}
 
   export NODE_PORT=$(kubectl get --namespace {{ .Release.Namespace }} -o jsonpath="{.spec.ports[0].nodePort}" services {{ template "mean.fullname" . }})
   export NODE_IP=$(kubectl get nodes --namespace {{ .Release.Namespace }} -o jsonpath="{.items[0].status.addresses[0].address}")
   echo "MEAN app URL: http://$NODE_IP:$NODE_PORT/"
 
-{{- else if contains "LoadBalancer" .Values.serviceType }}
+{{- else if contains "LoadBalancer" .Values.service.type }}
 
   NOTE: It may take a few minutes for the LoadBalancer IP to be available.
         Watch the status with: 'kubectl get svc -w {{ template "mean.fullname" . }} --namespace {{ .Release.Namespace }}'
 
   export SERVICE_IP=$(kubectl get svc --namespace {{ .Release.Namespace }} {{ template "mean.fullname" . }} -o jsonpath='{.status.loadBalancer.ingress[0].ip}')
   echo "MEAN app URL: http://$SERVICE_IP/"
 
-{{- else if contains "ClusterIP"  .Values.serviceType }}
+{{- else if contains "ClusterIP"  .Values.service.type }}
 
-  kubectl port-forward --namespace {{ .Release.Namespace }} svc/{{ template "mean.fullname" . }} 3000:3000
-  echo "MEAN app URL: http://127.0.0.1:3000/"
+  kubectl port-forward --namespace {{ .Release.Namespace }} svc/{{ template "mean.fullname" . }} {{ .Values.service.port  }}:{{ .Values.service.port }}
```

Comment:
It should be `{{ .Values.applicationPort }}` instead of 3000 no?

Reply:
No, It's addressing the service

Reply:
you're right